src/symfony/application/skeleton/composer.json.twig

```diff
@@ -0,0 +1,150 @@
+{
+    "name": "{{ @('workspace.name')|lower }}/symfony",
+    "type": "project",
+    "license": "proprietary",
+    "require": {
+        "php": ">=7.2.5",
+        "ext-ctype": "*",
+        "ext-iconv": "*",
+        "composer/package-versions-deprecated": "^1.11",
+        "doctrine/annotations": "^1.0",
+        "doctrine/doctrine-bundle": "^2.1",
+        "doctrine/doctrine-migrations-bundle": "^3.0",
+        "doctrine/orm": "^2.7",
+        "phpdocumentor/reflection-docblock": "^5.2",
+        "sensio/framework-extra-bundle": "^5.6",
+        "symfony/asset": "5.1.*",
+        "symfony/console": "5.1.*",
+        "symfony/dotenv": "5.1.*",
+        "symfony/expression-language": "5.1.*",
+        "symfony/flex": "^1.3.1",
+        "symfony/form": "5.1.*",
+        "symfony/framework-bundle": "5.1.*",
+        "symfony/http-client": "5.1.*",
+        "symfony/intl": "5.1.*",
+        "symfony/mailer": "5.1.*",
+        "symfony/mime": "5.1.*",
+        "symfony/monolog-bundle": "^3.1",
+        "symfony/notifier": "5.1.*",
+        "symfony/process": "5.1.*",
+        "symfony/property-access": "5.1.*",
+        "symfony/property-info": "5.1.*",
+        "symfony/security-bundle": "5.1.*",
+        "symfony/serializer": "5.1.*",
+        "symfony/string": "5.1.*",
+        "symfony/translation": "5.1.*",
+        "symfony/twig-bundle": "5.1.*",
+        "symfony/validator": "5.1.*",
+        "symfony/web-link": "5.1.*",
+        "symfony/yaml": "5.1.*",
+        "twig/extra-bundle": "^2.12|^3.0",
+        "twig/twig": "^2.12|^3.0"
```

Comment:
would it be worth only going for the minumum that symfony/flex sets up?

Reply:
Looks like the sample project we have chose v3.0.5 in https://github.com/inviqa/symfony-sample/blob/main/composer.lock#L6012, so I can adjust this to be the ^3.0 requirement?

Reply:
sorry I meant the whole dependency tree

Reply:
This is the composer.json (with additional dev dependencies added by me) generated by `composer create-project symfony/website-skeleton test-symfony`

We could use the "symfony/skeleton" but then won't have a website to test, or we can run the command upon applying the skeleton to a project like the spryker harness does, with a composer.json merge to add our development tools? Or something else?

Followed https://symfony.com/doc/current/setup.html#creating-symfony-applications

Reply:
symfony/skeleton includes the web bits as well

Reply:
Okay I'll re-generate it with just skeleton instead of web-skeleton

Reply:
considering not all apps use doctrine and it's a single command to install and configure with flex this can be documentation